src/components/whisperer/WhispererResponse.js

```
@@ -0,0 +1,198 @@
+  }
+
+  if (!result) return null;
+
+  const status = result.status;
+
+  if (status === "unauthorized" || status === "not_found") {
+    return (
+      <div
+        className={`mt-3 rounded-xl border border-white/10 bg-stone-900/60 p-4 text-sm text-stone-400 ${baseTransition} ${enterClasses}`}
+      >
+        Couldn't load this task. Refresh and try again.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)